imports/node-app/core-services/account/mutations/sendResetAccountPasswordEmail.js

```diff
@@ -2,7 +2,7 @@ import _ from "lodash";
 import SimpleSchema from "simpl-schema";
 import Logger from "@reactioncommerce/logger";
 import ReactionError from "@reactioncommerce/reaction-error";
-import generateVerificationTokenObject from "../util/generateVerificationTokenObject";
+import generateVerificationTokenObject from "@reactioncommerce/api-utils/generateVerificationTokenObject.js";
```

Comment:
Import order should be updated

```suggestion
import SimpleSchema from "simpl-schema";
import generateVerificationTokenObject from "@reactioncommerce/api-utils/generateVerificationTokenObject.js";
import Logger from "@reactioncommerce/logger";
import ReactionError from "@reactioncommerce/reaction-error";
```